### PR TITLE
DOCS-443 - Clean up API articles

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request does minor cleanup  of this section: https://help.sumologic.com/docs/api/

Note: It also adds a new hidden "Required role capabilities" section to articles in anticipation of the dev group eventually providing the role capabilities needed to run APIs (for [DOCS-413](https://sumologic.atlassian.net/browse/DOCS-413)).

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-443](https://sumologic.atlassian.net/browse/DOCS-443)

### CODE REVIEW

File: docs/api/access-keys.md [hunk 1]
```diff
@@ -3,16 +3,19 @@ id: access-keys
 title: Access Keys Management APIs
 sidebar_label: Access Keys
 description: Use HTTP endpoints to manage access keys.
-hide_table_of_contents: true
+hide_table_of_contents: false
```
Comment (kimsauce) [inline docs/api/access-keys.md:6]: `false` is the default, so you can just remove that `hide_table_of_contents: true` line.

```suggestion
```

Comment (jpipkin1) [reply docs/api/access-keys.md]: Thanks, @kimsauce. Fixed.

**Comment by jpipkin1** [comment]
@kimsauce, I made your suggested change. Could you take another look?

File: docs/api/archive-ingestion.md [hunk 2]
```diff
@@ -32,3 +33,13 @@ import ApiIntro from '../reuse/api-intro.md';
 | KR         | https://api.kr.sumologic.com/docs/#tag/archiveManagement  |
 | US1        | https://api.sumologic.com/docs/#tag/archiveManagement     |
 | US2        | https://api.us2.sumologic.com/docs/#tag/archiveManagement |
+
+<!-- ## Required role capabilities
+
+<ApiRoles/>
```
Comment (kimsauce) [inline docs/api/archive-ingestion.md:39]: Could you clarify why the `<ApiRoles/>` component is grayed out throughout this PR?

**Review by kimsauce — COMMENTED** [review]
Could you clarify why the `<ApiRoles/>` component is grayed out throughout this PR?

**Comment by jpipkin1** [comment]
@kimsauce, the `<ApiRoles/>` component is used in the hidden "Required role capabilities" sections, which I will unhide once I get the final list of role capabilities from Dev ([DOCS-413](https://sumologic.atlassian.net/browse/DOCS-413)).

**Review by kimsauce — APPROVED** [review]

**Pull request merged**